### ISSUE TITLE
Remember opened "sub-folders"

### ISSUE THREAD

**Issue by fredefox** [jaspervdj/profiteur#25]
In the HTML output I think it would be super useful if folding a parent folder with multiple open children would remember what children are opened. Would this be reasonably easy to add? Are you open for PRs? I might be interested in working on it. Would be great if you could point me in the right direction to get started.

Thanks for the tool!

**Comment by fredefox** [comment]
I see that the children are added to the DOM only when a folder is expanded and removed again when it is closed. So perhaps this is not so straight forward.

Is there a specific reason that not everything is added to the dom on the initial page load? I'm guessing there might be a great deal of HTML that would need to be generated for large profiler outputs...

**Comment by jaspervdj** [comment]
Yes, adding all of the children is not really possible for large enough profiler outputs, the browser will just end up freezing.

I don't see anything wrong with keeping the children around when closing the parent though.
